### ISSUE TITLE
Cannot train the model with SGD optimizer

### ISSUE THREAD

**Issue by chihchiehchen** [MIRACLE-Center/YOLO_Universal_Anatomical_Landmark_Detection#1]
Hello,

Thanks for sharing the code. After downloading the required dataset, the code performs well. However, when I try to modify the optimizer, for example, replace adam by sgd, the training process cannot converge even with smaller learning rate.

Could you give me some suggestions or some hints I shall take care of ?  In any case thanks a lot!

**Comment by heqin-zhu** [comment]
Hello, 

I am not sure about this problem. Maybe you can try a bigger learning rate and set `use_scheduler=False`.
I will re-train the model later. If I find something useful, I will reply to you immediately.
